ci/run-check.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# Enable warnings about unused extern crates
+# export RUSTFLAGS=" -W unused-extern-crates"
+rustc --version
+rustup --version
+cargo --version
+
+case $TARGET in
+  cargo-build)
+    cargo build --release "$@"
+    ;;
+
+  test-general)
+    cargo test --workspace --release --features runtime-benchmarks,try-runtime --exclude runtime-integration-tests
+    ;;
+
+  test-integration)
+    cargo test --release --package runtime-integration-tests --features fast-runtime
```

Comment:
@lemunozm feel free to add the debug flag you mentioned to these commands and see what it does. I'm not sure how to add it.

Reply:
Here it is: https://github.com/centrifuge/centrifuge-chain/pull/1606

Reply:
We finally abort the addition of debug until solving the integration test timing issue